airbyte-workers/src/main/java/io/airbyte/workers/DefaultGetSpecWorker.java

```diff
@@ -77,7 +77,7 @@ public DefaultGetSpecWorker(final IntegrationLauncher integrationLauncher) {
         // this.
         // retrieving spec should generally be instantaneous, but since docker images might not be pulled
         // it could take a while longer depending on internet conditions as well.
-        WorkerUtils.gentleClose(process, 2, TimeUnit.MINUTES);
+        WorkerUtils.gentleClose(process, 10, TimeUnit.MINUTES);
```

Comment:
what's the longest time we've heard someone take? i thought we said 30 min? 10 is fine if we think this covers all the cases we've seen.

Reply:
10m was finger-to-the-wind estimate, I don't see an issue with increasing the timeout to 30m. Updated.